composer.json

```diff
@@ -11,22 +11,25 @@
             "email": "dcramer@gmail.com"
         }
     ],
-    "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.1",
-        "phpunit/phpunit": "^4.8 || ^5.0",
-        "ext-openssl": "*",
-        "nokitakaze/testhttpserver": "^0.1.0",
-        "monolog/monolog": "~1.0"
-    },
     "require": {
-        "php": ">=5.5",
-        "ext-hash": "*",
+        "php": ">=5.6",
```

Comment:
I would write explicitly `^5.6 | ^7.0` to avoid any strange issue with future PHP versions, it's a good practice.

7.2 already may have some BC, but we will fix them; 8.x instead will require a new major version, I think.